### DISCUSSION TITLE
New track color presets for existing ride types

### DISCUSSION THREAD

**Discussion by SpartanFrederic104** [OpenRCT2/OpenRCT2#17785] [Feature requests]
karst's PR to add more new track pieces to the Flying Roller Coaster (#17782) (which is in draft at the time of this writing) also adds three new track color presets to the Flying Roller Coaster, several of them derived from real-world B&M Flying Coasters:
- Saturated red, saturated red, dark yellow
- Light blue, dark blue, icy blue (Manta at SeaWorld Orlando)
- Yellow, bright red, light orange (pre-2021 Tatsu at Six Flags Magic Mountain)

Since I'm bringing up the topic of track color presets, it might be a good idea to add more track color presets to existing ride types, either by deriving from real life coasters (e.g. Griffon's icy blue/icy blue/white for the Vertical Drop Roller Coaster) or by restoring lost track color presets from RCT1 and RCT2 vanilla (e.g. red/red/gray and bright purple/bright purple/bordeaux red for the Inverted Impulse Coaster, as shown by Loop-the-Loop's alternate track color schemes). This will especially be helpful for those ride types that have little in the way of track color presets (e.g. Vertical Drop Roller Coaster, Mini Suspended Coaster).
![Fred's Workbench 2022-08-11 08-17-52](https://user-images.githubusercontent.com/61885939/184199711-d54f898f-2ed7-4540-b625-db47ff2485af.png)

**Comment by 733737** [comment]
https://github.com/OpenRCT2/OpenRCT2/pull/17787
